create-a-new-policy.md

```diff
@@ -0,0 +1,2 @@
+1. Create a new folder in pkg/policies
```

Comment:
This doc can be updated with more specifics once we finish building and adding the new catalog-info check.